src/main/java/com/nha/abdm/wrapper/hip/hrp/dataTransfer/DataTransferInterface.java

```diff
@@ -0,0 +1,17 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.hip.hrp.dataTransfer;
+
+import com.nha.abdm.wrapper.common.responses.GatewayCallbackResponse;
+import com.nha.abdm.wrapper.hip.hrp.dataTransfer.responses.BundleResponseHIP;
+import com.nha.abdm.wrapper.hip.hrp.dataTransfer.responses.DataNotifyResponse;
+import com.nha.abdm.wrapper.hip.hrp.dataTransfer.responses.DataRequestResponse;
+
+public interface DataTransferInterface {
+  GatewayCallbackResponse DataOnNotifyCall(DataNotifyResponse data);
```

Comment:
Lets try to reuse the Pojo names similar to what ABDM has defined in their swagger.

Reply:
Made the POJO class names similar with ABDM schema